_update_deno.ts

```diff
@@ -66,7 +66,7 @@ await run(["git", "add", "*.dockerfile", "README.md", "example/Dockerfile"]);
 // Commit the changes
 await run(["git", "commit", "-m", `Rolling to deno ${newVersion}`]);
 
-// Push to the `hayd/deno-docker#autoroll`
+// Push to the `denoland/deno_docker#autoroll`
```

Comment:
headsup: the cron yml which uses this update script was never set up correctly (I think it needed the git user but didn't investigate - you could copy over those lines/secrets from rusty_v8 https://github.com/denoland/rusty_v8/blob/main/.github/workflows/update-v8.yml#L19

Reply:
thanks! i'll investigate that after merging this PR.